WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -118,7 +118,7 @@ public static int GetAnonymitySet(this Transaction me, int outputIndex)
 			int equalOutputs = me.GetIndistinguishableOutputs(includeSingle: true).Single(x => x.value == output.Value).count;
 			// 3. Anonymity set cannot be larger than the number of inputs.
 			var inputCount = me.Inputs.Count;
-			var anonSet = Math.Min(equalOutputs, inputCount);
+			var anonymitySet = Math.Min(equalOutputs, inputCount);
```

Comment:
I believe `anonSet` is used in a bunch of places, not sure if we want to change this...

Reply:
I think I changed it everywhere.